unity3d/Assets/Plugins/iOS/swrve_sdk.mm

```diff
@@ -1,5 +1,8 @@
 #import <CoreTelephony/CTTelephonyNetworkInfo.h>
 #import <CoreTelephony/CTCarrier.h>
+#if !defined(SWRVE_NO_IDFA)
```

Comment:
Can move to ifndef, if I believe gives more flexibility (but in this case it is useless).
Unity3D lets you define compile flags per files, at least the latest version.